Add known limitations page and workaround  for NHI issue

erickhun opened the pull request:
Those idea were from Philip that I thought good: 

We should be clear about the limitation of the card to avoid any new comers to be "surprised"
Also added a draft (not public yet) on the upcoming changes. 

@fifieldt how does that feel to you?

fifieldt commented:
Thanks for tagging, thinking.

fifieldt commented:
Hi,

I think this could be a good idea, but we're not there yet.

Why? Early on, when people are all very excited about a project, there is a temptation to split content amoungst many different buckets. We gleefully assume that one day we'll have entire vibrant sections under each new title we create. Though that may turn out to be true, the trap with the approach is: during the early stages of the project, the content looks very thin and the site "empty".

I think for now, what would be best is to expand the existing health insurance question with the 'solution'. For now, news related to the New Economic Immigration Bill can probably be listed in news when it occurs, since its implementation is on indeterminate timeframe (and not guaranteed).

erickhun commented:
@fifieldt agree , how does that feel now? 

- moved it in the appropriate section
- switch to draft so it's not published, but we can reuse-later. ( I still have  a feeling that it could be useful for new applicant to know what to expect)

I've also split the `Am I allowed to leave the country during the six-month period wait?` Q, so it's not lost in an other answer

fifieldt commented:
checking

fifieldt commented:
I think this is good to go now.
